localisation/00_memewc_united_localisation.csv

```diff
@@ -1,6 +1,18 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 RULE_GROUP_WARCRAFT_ARTIFACTS;WarCraft - Artifacts;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Physical Weapons;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use some physical weapons.;;;;;;;;;;;;;x
```

Comment:
Removed `some`, because I don't _think_ its necessary. 
```suggestion
RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use physical weapons.;;;;;;;;;;;;;x
```